torch/csrc/jit/symbolic_script.cpp

```diff
@@ -1380,12 +1383,23 @@ c10::optional<GradientPair> gradientInfoForSchema(
     //    in its argument, all scalar arg should already be passed with float
     //    value since scalar/int aren't differentiable either way.
     //
-    c10::ReplaceAll(schema_str, "Scalar", "float");
     // For debugging AD change:
     // std::cout << "Looking for " << schema_str << std::endl;
 
     auto sym_script_it = schema_to_graphs.find(schema_str);
 
+    if (sym_script_it == schema_to_graphs.end()) {
```

Comment:
Yea this `if` should be dropped before merging.

Reply:
So I removed it after double checking by means of calling `sig(schema_string)` and seeing whether it throws here:
https://github.com/pytorch/pytorch/blob/3e0b46b6d132c22af84769a6df1ef8d1f785dbe2/torch/csrc/jit/symbolic_script.cpp#L1351-L1353